### PR TITLE
feat(cli): if no out or json cmd, put out to ./docs

### LINKED ISSUE #746: default out option
Typedoc requires either an `--out` or `--json` argument to run. It should have a reasonable default option (e.g. `--out docs`) to make getting started easier.

Suggested by @ice1080

### PR DESCRIPTION
**Pull request by artentica**
I not really happy with this solution but idk how do another way.
If you have an idea i'll be happy to implement it

Closes #746

### CODE REVIEW

File: src/lib/cli.ts [hunk 1]
```diff
@@ -74,6 +71,11 @@ export class CliApplication extends Application {
                 if (this.json) {
                     this.generateJson(project, this.json);
                 }
+                if (!this.out && !this.json) {
+                    this.logger.log("You must either specify the 'out' or 'json' option.");
```
Comment (aciccarello) [inline src/lib/cli.ts]: If we are automatically setting a default "must" sounds strong here.
```suggestion
                    this.logger.log("No 'out' or 'json' option has been set");
```

File: src/lib/cli.ts [hunk 2]
```diff
@@ -74,6 +71,11 @@ export class CliApplication extends Application {
...
+                    this.logger.log("The './docs' directory have been set in default output");
```
Comment (aciccarello) [inline src/lib/cli.ts]:
```suggestion
                    this.logger.log("The './docs' directory has be set as the output location by default");
```

Comment (artentica) [reply src/lib/cli.ts]: Of course, I correct that and repush
Sorry for the bad grammar :/
Thx for your suggestions =)

**Comment by artentica** [comment]
I accept the modification (directly on github (didn't know it was possible ^^))

**Comment by artentica** [comment]
I'm not sure i ask: I cannot merge even if you (@aciccarello) approved no ?

**Comment by aciccarello** [comment]
I'm leaving it open in case someone else wanted to chime in. I'll merge before the next release.

**Pull request merged**